ModelCatalogueCorePlugin/test/integration/x/org/modelcatalogue/core/dataarchitect/UmljServiceISpec.groovy

```diff
@@ -2,23 +2,24 @@ package x.org.modelcatalogue.core.dataarchitect
 import org.modelcatalogue.core.AbstractIntegrationSpec
 import org.modelcatalogue.core.DataModel
 import org.modelcatalogue.core.DataClass
+import spock.lang.Ignore
 
 class UmljServiceISpec extends AbstractIntegrationSpec {
 
     def umljService, initCatalogueService, catalogueBuilder
 
-//    @Ignore
+    @Ignore
```

Comment:
why is this ignored? it run fine but for some reason it can't run as a part of other integration specs (don't know exactly why :-() but if it run as a part of "slow" tests, it's fine.